driftbase/api/runconfigs.py

```diff
@@ -30,6 +27,16 @@ def drift_init_extension(app, api, **kwargs):
     endpoints.init_app(app)
 
 
+class RunConfigsPostSchema(ma.Schema):
+    name = ma.fields.String()
+    repository = ma.fields.String()
+    ref = ma.fields.String()
+    build = ma.fields.String()
+    num_processes = ma.fields.Integer(required=False)

```

Comment:
isn't required False by default?

Reply:
Yes, I had read that wrong initially and set everything to required=False. Must have missed this when inverting that. No harm, but to keep style consistent I'll remove it.

Reply:
"name", "repository", "ref" and "build" should then have required=True to match the earlier simple_schema_request.